samples/vboxwrapper/vboxwrapper.cpp

```diff
@@ -483,6 +486,9 @@ int main(int argc, char** argv) {
         if ((5 == vbox_major) && (1 <= vbox_minor)) {
```

Comment:
Should this test for equality only? I think that otherwise vboxwrapper will try to create a VM first with 5.1 interface (which might not be a huge problem) if VirtualBox 5.2 is installed.

Reply:
Thanks for catching that.